Vinyl/Turntable.swift

```diff
@@ -85,32 +86,30 @@ public final class Turntable: NSURLSession {
     }
 
     public override var delegate: NSURLSessionDelegate? {
-        return nil
+        return _delegate
     }
 }
 
 // MARK: - NSURLSession methods
 
 extension Turntable {
+
+    public override func dataTaskWithURL(url: NSURL) -> NSURLSessionDataTask {
+        let request = NSURLRequest(URL: url)
+        return dataTaskWithRequest(request)
+    }
     
```

Comment:
It's preferable to add a default value to the completionHandler: 

```
dataTaskWithURL(url: NSURL, completionHandler: (NSData?, NSURLResponse?, NSError?) -> Void) -> NSURLSessionDataTask
```
Than to clutter the api with extra methods.